logging.go

```diff
@@ -22,6 +22,10 @@ type Logger struct {
 func makeLogger(url string) Logger {
 	channel := make(chan LogMessage, 256)
 	go func() {
+		if url == "" {
+			for range channel {
+			}
```

Comment:
While not normally reachable, it might be a better idea to add a `return` after this loop, just in case a future update closes this channel.